release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -440,6 +440,15 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-insights-operator-enhancements_{context}"]
 === Insights Operator
 
+
+[id="ocp-release-notes-insights-operator-runtime-extractor_{context}"]
+==== Insights Runtime Extractor is generally available
+
+In {product-title} 4.18, the Insights Operator introduced the _Insights Runtime Extractor_ workload data collection feature as a Technology Preview feature to help Red{nbsp}Hat better understand the workload of your containers. 
+Now, in version 4.19, the feature is generally available. 
+The Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat.
+Red{nbsp}Hat uses the data to gain insights that can help you make investment decisions that drive and optimize your use of  {product-title} containers.
```

Comment:
I think I don't understand this last sentence here. Red Hat uses the data to gain insights, but when I am reading this as a customer I think it wouldn't help me to make any investment decision. My understanding is that the data is solely for internal use. am I missing something?

Reply:
Thanks, @tremes 👍 
Line 450 was inherited from previous descriptions of the Runtime Extractor.
For this 4.19 GA release, I have now removed the sentence on line>450. It is a bit fluffy and "salesy" for a release note. If you'd like anything added, let me know. Otherwise, I will prepare this note for publishing on GA day.

Do we have any other release notes to add to the Insights Operator section for 4.19GA? Checking as the primary doc freeze deadline for release notes is this week. Thanks

Reply:
@michelle-purcell Thank you! I am fine with the sentence removal. I moved to the Observability analytics team so for some other Insights Operator release notes please contact @ncaak.

Reply:
Jumping in. @michelle-purcell I checked the changes we added to the 4.19 and confirm we don't need any more release notes for this version.

Thanks.

Reply:
Thanks for all your help, @tremes. Best of luck in the new team also ⭐

Reply:
@ncaak - Thank you for reviewing and confirming too 👍